sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveUserProvider.java

```diff
@@ -0,0 +1,16 @@
+package io.sentry.spring.reactive;
+
+import io.sentry.protocol.User;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.web.server.ServerWebExchange;
+
+/**
+ * Provides user information that's set on {@link io.sentry.SentryEvent}.
+ *
+ * <p>Out of the box Spring integration configures single {@link SentryReactiveUserProvider}.
+ */
+@FunctionalInterface
+public interface SentryReactiveUserProvider {
+  @Nullable
+  User provideUser(ServerWebExchange request);
```

Comment:
In case of Webflux, we need to return here `Mono<User>`, as user object may need a call to database or external service.

Reply:
Thanks for the suggestion @maciejwalkowiak, I'll change that.